From 62924413c9e42d1158216def76d14554529b016b Mon Sep 17 00:00:00 2001
From: "Rafael G. Martins" <rafael@rafaelmartins.eng.br>
Date: Mon, 15 Jul 2024 02:05:20 +0200
Subject: [PATCH 2/3] chromium: add conda paths to include_dirs

cmake won't properly pass the include dirs to gn. we need to help it
find our headers
---
 src/3rdparty/chromium/build/config/compiler/BUILD.gn | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/3rdparty/chromium/build/config/compiler/BUILD.gn b/src/3rdparty/chromium/build/config/compiler/BUILD.gn
index bcc97108b..4efb923e3 100644
--- a/src/3rdparty/chromium/build/config/compiler/BUILD.gn
+++ b/src/3rdparty/chromium/build/config/compiler/BUILD.gn
@@ -247,6 +247,10 @@ assert(!(llvm_force_head_revision && use_goma),
 assert(!(llvm_force_head_revision && use_remoteexec),
        "can't use rbe with trunk clang")
 
+conda_build_prefix = getenv("BUILD_PREFIX")
+conda_host = getenv("HOST")
+conda_build_include = conda_build_prefix + "/" + conda_host + "/sysroot/usr/include"
+
 # default_include_dirs ---------------------------------------------------------
 #
 # This is a separate config so that third_party code (which would not use the
@@ -254,6 +258,7 @@ assert(!(llvm_force_head_revision && use_remoteexec),
 # this and specify their own include paths.
 config("default_include_dirs") {
   include_dirs = [
+    conda_build_include,
     root_gen_dir,
     "//",
   ]
-- 
2.43.0

